src/Editor/EditorUtils.js

```diff
@@ -27,12 +27,30 @@ export const displayTextWithMentions = (inputText, formatMentionNode) => {
             });
         }else{
             formattedText.push(retLine);
-        }            
+        }
         formattedText.push("\n");
-    });    
+    });
     return formattedText;
 }
 
+export const formattedTextToPlain = (inputText) => {
+  let output = '';
+  const mentions = EU.findMentions(inputText)
+  if(mentions.length  === 0) {
+    return inputText
+  }
+  let lastIdx = 0;
+  mentions.forEach(mention => {
```

Comment:
Use a regular `for`, as this code actually depends and modifies external variables, so it's easier to understand.